master/security/protect-hosts.md

```diff
@@ -0,0 +1,182 @@
+
+#### Failsafe rules
+
+It is easy to inadvertently cut all host connectivity because of non-existent or misconfigured network policy. To avoid this, Calico provides failsafe rules with default/configurable ports that are open on all host endpoints.
+
+#### Default behavior of workload to host traffic
+
+By default, Calico blocks all connections from a workload to its local host.You can control whether connections from a workload endpoint to its local host are dropped, returned, or accepted using a simple parameter.
+
+Calico allows all connections from processes running on the host to guest workloads on the host. This allows host processes to run health checks and debug guest workloads.
+
+#### Default behavior of external traffic to/from host
+
+If a host endpoint is added and network policy is not in place, the Calico default is to deny traffic to/from that endpoint (except for traffic allowed by failsafe rules). For host endpoints, Calico blocks traffic only to/from interfaces that it’s been explicitly told about in network policy. Traffic to/from other interfaces is ignored.
+
+#### Other host protection
+
+For the sake of consistency in the Calico control plane, you may wonder about the following use cases.
+
+**Does Calico protect a local host from workloads?** 
+Yes. DefaultEndpointToHostAction controls whether or not workloads can acesss their local host.
+
+**Does Calico protect a workload from the host it is running on?** 
+No. Calico allows connections the host makes to the workloads running on that host. Some orchestrators like Kubernetes depend on this connectivity for health checking the workload. Moreover, processes running on the local host are often privileged enough to override local Calico policy. Be very cautious with the processes you allow to run in the host's root network namespace.
+
+### Before you begin...
+
+On hosts that you want to secure with network policy, [install Calico]({{site.baseurl}}/{{page.version}}/getting-started/bare-metal/installation/binary-mgr) and [install and run Felix]({{site.baseurl}}/{{page.version}}/getting-started/bare-metal/installation/).
+
+### How to
+
+- [Avoid accidentally cutting all host connectivity ](#avoid-accidentally-cutting-all-host-connectivity)
+- [Use policy to restrict host traffic](#use-policy-to-restrict-host-traffic)
+- [Control default behavior of workload endpoint to host traffic](#control-default-behavior-of-workload-endpoint-to-host-traffic)
+
+#### Avoid accidentally cutting all host connectivity
+
+To avoid inadvertently cut off all host connectivity because of non-existent or misconfigured network policy, Calico uses failsafe rules that open specific ports on all host endpoints. 
+
+Review the following table to determine if the defaults work for your implementation. If not, change the default ports using the parameters, **FailsafeInboundHostPorts** and **FailsafeOutboundHostPorts** in [Configuring Felix]().
```

Comment:
Missing a link here for `Configuring Felix`

Reply:
Fixed.